src/APL/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from "./apl";
 export * from "./env-apl";
 export * from "./file-apl";
+export * from "./redis";
```

Comment:
@JannikZed let's avoid that, now importing any other APL will evaluate redis, this causes issues, e.g. next is poor at tree shaking, possibly fail because it can't import redis sdk

Please check package.json and add `exports` key with new path, like vercel-kv. For example 

```
    "./APL/redis": {
      "types": "./APL/redis/index.d.ts",
      "import": "./APL/redis/index.mjs",
      "require": "./APL/redis/index.js"
    },
```

also new entry point will be needed

https://github.com/saleor/app-sdk/blob/main/tsup.config.ts#L13

Reply:
@IKarbowiak alright, check it out, if that works for you.
Is there a very easy way to test also if the package got build correctly and we can actually use the different APLs? are you spinning up an example saleor app and installing it locally or how are you doing that?